packages/solarwinds-apm/package.json

```diff
@@ -45,23 +45,22 @@
     "test": "swtest -p test/tsconfig.json -c src"
   },
   "dependencies": {
-    "@opentelemetry/auto-instrumentations-node": "^0.39.2",
     "@opentelemetry/core": "~1.17.0",
     "@opentelemetry/instrumentation": "~0.43.0",
     "@opentelemetry/resources": "~1.17.0",
     "@opentelemetry/sdk-metrics": "~1.17.0",
     "@opentelemetry/sdk-trace-node": "~1.17.0",
     "@opentelemetry/semantic-conventions": "~1.17.0",
     "@solarwinds-apm/bindings": "workspace:^",
+    "@solarwinds-apm/instrumentations": "workspace:^",
+    "@solarwinds-apm/module": "workspace:^",
     "@solarwinds-apm/sdk": "workspace:^",
-    "zod": "^3.22.2"
+    "zod": "^3.22.4"
   },
   "peerDependencies": {
     "@opentelemetry/api": "^1.3.0",
-    "@opentelemetry/exporter-metrics-otlp-grpc": "~0.43.0",
-    "@opentelemetry/exporter-trace-otlp-grpc": "~0.43.0",
-    "json5": ">=1.0.0",
-    "ts-node": ">=9.0.0"
+    "@opentelemetry/exporter-metrics-otlp-grpc": ">=0.26.0",
```

Comment:
Ooh what's special about version 0.26.0?

Reply:
It's the first version that supports `@opentelemetry/api@1.0.0`. Now that you mention it since we depend on `1.3.0` I should probably bump it

Reply:
Bumped to the appropriate versions for 1.3.0